Source/Details/ASDataController.mm

```diff
@@ -617,6 +617,28 @@ - (void)_insertElementsIntoMap:(ASMutableElementMap *)map
 
 #pragma mark - Relayout
 
+- (void)relayoutNodes:(id<NSFastEnumeration>)nodes nodesSizeChanged:(NSMutableArray **)nodesSizesChanged
```

Comment:
We should use either `(out NSArray **)` or `(NSMutableArray *)` for the second param.